test/common.coffee

```diff
@@ -6,6 +6,7 @@ config =
   port: 27017
   db: '_mm'
   collection: '_migrations'
+  timeout: 10
```

Comment:
Yeah that should be fine. What I meant is that it influences all the test cases, so we need some timeout that will account for all the real operations in the rest of the tests.